### PR TITLE
Added FreeBSD section for Binary download

### LINKED ISSUE #1681: Add FreeBSD to binary-only distribution page
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Context

https://github.com/sensu/sensu-enterprise-go/pull/602 adds binary-only artifacts for FreeBSD on 386 and amd64 architectures.

### PR DESCRIPTION
**Pull request by Blue0ctober**
still requires links to packages

## Description

Adds section and download links for FreeBSD binary packages

## Motivation and Context

Closes #1681

### CODE REVIEW

File: content/sensu-go/5.12/installation/verify.md
Comment (Blue0ctober) [inline content/sensu-go/5.12/installation/verify.md]:
```suggestion
{{< platformBlock "Freebsd" >}}
```

**Comment by cwjohnston** [comment]
Looks good to me so far. We can't merge it until we have the final artifact URLs, so marking as WIP.

**Comment by Blue0ctober** [comment]
@amdprophet can you link us the url please, once available?

File: content/sensu-go/5.12/installation/verify.md
Comment (cwjohnston) [inline content/sensu-go/5.12/installation/verify.md]:
```suggestion
[32]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.12.0/sensu-go_5.12.0_freebsd_amd64.tar
```

File: content/sensu-go/5.12/installation/verify.md
Comment (cwjohnston) [inline content/sensu-go/5.12/installation/verify.md]:
```suggestion
[33]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.12.0/sensu-go_5.12.0_freebsd_amd64.zip
```

File: content/sensu-go/5.12/installation/verify.md
Comment (cwjohnston) [inline content/sensu-go/5.12/installation/verify.md]:
```suggestion
[34]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.12.0/sensu-go_5.12.0_freebsd_386.tar.gz
```

File: content/sensu-go/5.12/installation/verify.md
Comment (cwjohnston) [inline content/sensu-go/5.12/installation/verify.md]:
```suggestion
[35]: https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.12.0/sensu-go_5.12.0_freebsd_386.zip
```

File: content/sensu-go/5.12/installation/verify.md
Comment (cwjohnston) [inline content/sensu-go/5.12/installation/verify.md]: In each link `sensu-enterprise-go` prefix needs to become `sensu-go`

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.12/installation/verify.md
Comment (cwjohnston) [inline content/sensu-go/5.12/installation/verify.md]:
```suggestion
curl -LO https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.12.0/sensu-go_5.12.0_freebsd_amd64.tar.gz
```

File: content/sensu-go/5.12/installation/verify.md
Comment (cwjohnston) [inline content/sensu-go/5.12/installation/verify.md]:
```suggestion
sha512sum sensu-go_5.12.0_freebsd_amd64.tar.gz
```

**Review by cwjohnston — APPROVED** [review]
Approved contingent on the two pending suggestions being applied.

**Pull request merged**
